Core/Battler.gd

```diff
@@ -24,6 +23,7 @@ func _ready() -> void:
 		
 	PhaseManager.on_phase_changed.connect(_on_phase_changed)
 	CardManager.on_card_container_initialized.connect(_on_card_container_initialized)
+	CardManager.on_card_action_finished.connect(_handle_deaths.unbind(1))
```

Comment:
I am reusing the _"on_card_action_finished"_ signal which is defined in CardManager and used for other purposes. This also comes with an argument for a card that's being played. I just need to know that a card has been played and do not care for which card at this moment, so I opted for unbinding it rather then adding an argument to the __handle_deaths_ function that would never be used. 

I could also make my own signal without an argument and use it for this if you think it would be better to keep the signals separate for some reason. I didn't want to create a signal which would basically perfectly shadow the _on_card_action_finished_ signal we already had as I didn't see a reason to. Let me know if you can think of any concerns regarding that

Reply:
No no it's fine as is, I just wanted to make sure to understand